recipe/meta.yaml

```diff
@@ -19,17 +19,19 @@ build:
 requirements:
   host:
     - python
+    - setuptools
     - pip
   run:
     - python
 
 test:
   commands:
-    - futurize -h
-    - pasteurize -h
-
+    - pip check
```

Comment:
The `futurize -h` and `pasteurize -h` commands should still be kept. We (probably) just don't need to test the import of their corresponding `lib*` modules.